packages/vaadin-grid/src/vaadin-grid-data-provider-mixin.js

```diff
@@ -168,6 +168,10 @@ export const DataProviderMixin = (superClass) =>
           }
         },
 
+        _hasData: {
```

Comment:
nit: JSDoc is missing, should it be `@protected` or `@private`?

Reply:
Marked as protected